README.md

```diff
@@ -53,7 +53,7 @@ The script run by `npm run build:protos` can be found at `scripts/gen_grpc_web_p
 The current TypeScript proto library was generated with `protoc-gen-grpc-web` version 1.0.7 with
 `protoc` version 3.10.1.
 
-The Protocol Buffers in `proto/ml_metadata` are taken from version 0.22.1 of the `ml_metadata` proto
+The Protocol Buffers in `proto/ml_metadata` are taken from the master branch(v0.25.1) of the `ml_metadata` proto
```

Comment:
Did you take it from master branch or v0.25.1?
We should always use a released tag to make sure it's well-tested.

Reply:
Added script on replacing protos. PTAL!